brickflow_plugins/databricks/uc_to_snowflake_operator.py

```diff
@@ -1,9 +1,12 @@
 import logging
-import snowflake.connector
-import os
-from http import HTTPStatus
 
-from brickflow.context import ctx
+try:
+    import snowflake.connector
+except ImportError:
+    raise ImportError(
+        "You must install snowflake library to use run snowflake plugins, "
+        "please try pip install snowflake-connector-python"
```

Comment:
this should only be required on the cluster that is using this operator. 

@wf.task(libraries="...")